task/export.go

```diff
@@ -61,24 +63,75 @@ func TaskExport(tctx *TaskContext) (*TaskHandlerOutput, error) {
 	defer cancel()
 	tctx = tctx.WithContext(ctx)
 	content := tctx.Progress.TrackReader(file.Body, size, 1)
-	output, err := uploadFile(tctx, asset, content)
+
+	name := "asset-" + asset.PlaybackID
+	contentType := "video/" + asset.VideoSpec.Format
+	cid, internalMetadata, err := uploadFile(tctx, customParams, ipfsParams, name, content, contentType)
+	if err != nil {
+		return nil, err
+	}
+	nftCid, err := uploadNftMetadata(tctx, ipfsParams, cid)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
We probably need to check if `ipfsParams != nil` before calling this, just to still support that custom export one (not critical, not sure if anyone has ever used that tbh)

Reply:
Right, good catch. Updated in 9f5ed61c0588d6a81c41ac5524b176c9b48cbf95